### PR TITLE
Add Simplified Chinese translation

### PR DESCRIPTION
**Pull request by erithair**

### CODE REVIEW

**Comment by janl** [comment]
Thank you! — Now looking for another native speaker to give this a review and +1: https://twitter.com/janl/status/582627670317539328

**Comment by erithair** [comment]
Just noticed the discussion in #25, agree "Be excellent with each other" is more positive, though personally I prefer "Don't be a jerk", 
I think "互敬互爱“ is more likely used between family, especial couples. How about something like "友好待人” or “友善待人”

**Comment by muggleyoung** [comment]
友善待人 is much better :)

**Comment by erithair** [comment]
Translated "tl;dr: Be excellent with each other" to "简而言之：友善待人"

**Comment by remy** [comment]
Did this ever get signed off?

File: index-cn.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
```
Comment (Artoria2e5) [inline index-cn.html]: Do some lang=zh-Hans-CN here. Works even better than these hard-coded sans fonts.

File: index-cn.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+.footer p {
+  text-align: left;
+}
+</style>
+</head>
+<body>
+  <h1>活动守则</h1>
+
+  <p>所有参与本次会议的出席者，演讲者，赞助人，志愿者均应同意遵守下列行为规范。活动组织者将在活动过程中执行这些规范。我们期望所有参与者共同合作，提供一个安全的环境给大家。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: 提供一个安全的环境给大家 => 为大家提供一个安全的环境 (move-for-forward)

File: index-cn.html [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+<p><em>简而言之：友善待人</em></p>
+
+<h2>需要帮助？</h2>
+
+<p>在寄给您的 email 中可以找到我们的联络方式。</p>
+
+<h2>简短版</h2>
+
+<p>本会议致力于提供一个无骚扰的会议环境给每一位参与者，无论其性别、年龄、性倾向、残疾、外貌、身材、种族或信仰。我们绝不容忍任何人对会议参与者以任何形式的骚扰。在任何的会议场地中使用含有性的言语及意向在本会议中是不恰当的，包括在讲座中、工作坊、派对上、Twitter 上或任何的网络媒体中。与会者若违反规定，将受到制裁或被驱逐离开会议并<em>不予退款</em>，处置由活动组织者裁定。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: * 为所有与会者提供一个无骚扰的会议环境 (move-for-forward)
* 性倾向 => 性向 (align-chars: 2)
* 在任何的……不恰当的，包括……网络媒体中。=> 不应在任何的……，包括……。 (doing-to-shouldn't)

File: index-cn.html [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+<h2>完整版</h2>
```
Comment (Artoria2e5) [inline index-cn.html]: Consider retaining the quick/not so quick contrast. Not as interesting to say "short/full".

File: index-cn.html [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>骚扰包括关于性别、性倾向、残疾、生理外观、身材、种族、宗教的言语冒犯，在公共场所展示带有性含意的图片，故意恐吓、尾随、跟踪、骚扰摄影或录音，持续以言语或其他形式干扰，不适当的肢体接触，以及不受欢迎的性关注。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: 关于 => 有关

File: index-cn.html [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>当与会者要求停止骚扰行为时，请立即停止相关的骚扰行为。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: Possible mistranslation:
* en: if participant `p` is asked to stop harassing by *anyone else*, then `p` should cease the behavior.
* current: if someone else `p` is asked by a *participant* to stop harassing....

Suggested fix: 与会者 => 有人

File: index-cn.html [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>反骚扰的政策也适用于会议的赞助者。特别地，赞助者不应该使用任何带有性含意的图片、行为、或任何形式的内容。工作人员（包括志愿者）不应使用任何带有性含意的衣服、制服、装扮，或以其他任何方式营造带有性含意的环境。</p>
+
+<p>如果参加者骚扰或挑衅他人，活动组织者可以采取各种他们认为适当的处置，包括警告或驱逐肇事者，并不予退款。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: Inconsistent translation for participant. Unify 参加者 to 与会者.

File: index-cn.html [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>如果您被骚扰，或者发现有人被骚扰，或有任何其他疑虑，请立即通知活动工作人员。工作人员会穿着可识别的服饰。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: 可识别的 => 可明确辨识的

File: index-cn.html [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+<p>我们重视每一位与会者的参与。会场工作人员很乐意协助与会者联络饭店、会场的保全人员或地方执法单位以提供护送服务或任何能够让受到骚扰的与会者在会议期间感到安全的协助。</p>
```
Comment (Artoria2e5) [inline index-cn.html]: 保全人员 => 保安人员

Please proofread for zh-tw-isms before you label things as zh-cn.

Comment (Artoria2e5) [inline index-cn.html]: Sentence possibly too long. Can we slice 任何能够让受到骚扰的与会者在会议期间感到安全的协助 apart by a bit?

File: index-cn.html [hunk 10]
```diff
@@ -0,0 +1,104 @@
...
+  font-size: 18px;
+  text-align: justify;
+}
+
+a {
+  color: blue;
+}
+
+em {
+  font-weight: 400;
```
Comment (Artoria2e5) [inline index-cn.html]: Override the user agent stylesheet to enforce non-italic presentation here. zh text should not be presented in oblique. See replacement at https://zh.wikipedia.org/wiki/Template:Zh-em.

File: index-cn.html [hunk 11]
```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf8">
+<title>活动守则</title>
+<style>
+* {
+  font-family: "Microsoft YaHei",  "微软雅黑体", "Helvetica Neue", Helvetica, Arial, sans-serif;
```
Comment (Artoria2e5) [inline index-cn.html]: "微软雅黑体" is redundant -- merely an alias to "Microsoft YaHei". By the way, isn't it called 微软雅黑?

Comment (Artoria2e5) [inline index-cn.html]: Throwing my usual objection to hardcoding Chinese sans-serifs here too: It's usually better for the user agent to decide what to use for the user's browser. (Yes, Microsoft Edge does not use that awful SimSun serif thing for sans-serif like IE, so you will be fine, really.) Hardcoding fonts that don't have multiple weights while invoking a lighter weight of 200 is pretty bad.

File: index-cn.html [hunk 12]
```diff
@@ -0,0 +1,104 @@
...
+<p>我们重视每一位与会者的参与。会场工作人员很乐意协助与会者联络饭店、会场的保全人员或地方执法单位以提供护送服务或任何能够让受到骚扰的与会者在会议期间感到安全的协助。</p>
+
+<p>我们希望所有的与会者不论在会场、工作坊、及任何与会议相关的社交活动上都能够遵守上述规则。</p>
+
+<div class="footer">
+<p><small><em>原文出处：<a href="http://2012.jsconf.us/#/about">http://2012.jsconf.us/#/about</a> &amp; <a href="http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy">The Ada Initiative</a><br>
+
+协助翻译与改进：<a href="https://github.com/leftlogic/confcodeofconduct.com">http://github.com/leftlogic/confcodeofconduct.com</a><br>
+
+文件以 <a rel="license" href="http://creativecommons.org/licenses/by/3.0/deed.en_US">Creative Commons Attribution 3.0 Unported License</a> 授权</em></small>
```
Comment (Artoria2e5) [inline index-cn.html]: Can use some HTTPS here. Might need a global change.

**Comment by MattiSG** [comment]
Thank you so much @Artoria2e5! 

@erithair are you still available to handle this review? 😃

**Comment by erithair** [comment]
@Artoria2e5 Thanks for your review!
@MattiSG Sorry I missed this. I switched to another GitHub account since 2017 due to personal reason. I'll  handle the review as soon as I can.

**Comment by MattiSG** [comment]
Hi @erithair! It's been a month since you mentioned wanting to handle the review 🙂 This is just a simple reminder that it would be great if this got done! 😉
Also, please feel free to give us your new GitHub account so that I can mention it here as well.

File: index-cn.html (hunk 3 above)
Comment (erithair) [reply index-cn.html]: I kept "性倾向" as "性向" may have more wide meanings than "性情向".

File: index-cn.html (hunk 4 above)
Comment (erithair) [reply index-cn.html]: I don't fully understand what needs to be done here. Can you be more specific please?

File: index-cn.html (hunk 6 above)
Comment (erithair) [reply index-cn.html]: Translated to: 当与会者被要求停止骚扰行为时

File: index-cn.html (hunk 9 above)
Comment (erithair) [reply index-cn.html]: > Sentence possibly too long. Can we slice 任何能够让受到骚扰的与会者在会议期间感到安全的协助 apart by a bit?

How about `会场的保安人员或地方执法单位以提供护送服务，或协助保障任何受到骚扰的与会者在会议期间的安全。`?

**Comment by erithair** [comment]
@MattiSG Apologize for your long wait. I've applied the most of suggestions by @Artoria2e5 , except one or two might still need discussion. 
And please feel free to mention my new GitHub account @ug0 so I'll be notified.

**Comment by MattiSG** [comment]
Great, thanks @ug0!

We'll see if @Artoria2e5 is available to keep the discussion going. If not, I'll merge as such next week 🙂

**Comment by Artoria2e5** [comment]
:+1:

**Comment by MattiSG** [comment]
Lovely, thanks to you both! 🎉

**Pull request merged**